egs/zeroth_korean/s5/local/updateSegmentation.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
please call this update_segmentation.sh, for consistency in naming.

Reply:
changed it